include/GafferImage/ImageReader.h

```diff
@@ -97,6 +97,9 @@ class GAFFERIMAGE_API ImageReader : public ImageNode
 		Gaffer::IntPlug *missingFrameModePlug();
 		const Gaffer::IntPlug *missingFrameModePlug() const;
 
+		Gaffer::IntPlug *channelNamingPlug();
+		const Gaffer::IntPlug *channelNamingPlug() const;
```

Comment:
I'm hoping that people aren't going to need to touch this one much, so maybe it belongs at the bottom of the list of plugs rather than in the middle? I wonder if "interpretation" rather than "naming" describes what we're doing a bit better, and sounds more "heuristicky"? Not sure...see comments about `ImageWriter.namingConvention`.